tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -129,7 +129,7 @@ class DatasetParams {
                 std::vector<PartialTensorShape> output_shapes, string node_name,
                 DatasetParamsType type);
 
-  ~DatasetParams() {}
+  virtual ~DatasetParams() {}
```

Comment:
Add the virtual destructor for the base class, but did not add the destructor for the derived class in order to make the code concise (also the destructor does not do anything here). But please feel free to let me know if it is a good practice to add the destructor for the derived classes.